articles/api/authentication/_passwordless.md

```diff
@@ -208,22 +208,11 @@ Once you have a verification code, use this endpoint to login the user with thei
 | <dfn data-key="scope">`scope`</dfn> | Use `openid` to get an ID Token, or `openid profile email` to also include user profile information in the ID Token. |
 | `redirect_uri` <br/><span class="label label-danger">Required</span> | A callback URL that has been registered with your application's **Allowed Callback URLs**. |
 
-
-### Test with Authentication API Debugger
```

Comment:
It's great to see references to the Authentication API Debugger removed 🙂

Reply:
Yes! I haven't heard if the Debugger has been fully deprecated or not, so I have a single reference in the intro for it.

Reply:
It's not fully deprecated. There is an intention to deprecate it, but not a plan AFAIK.
What you did looks perfect to me: maintaining a reference to it but reducing the doc coverage to discourage new people from using it.